core/modules/system/system.install

```diff
@@ -3632,6 +3643,14 @@ function system_update_1097() {
   }
 }
 
+
+/**
+ * Dummy update.
+ */
+function system_update_1099() {
```

Comment:
Is that an oversight, or just there for more convenient local testing? :wink:

Reply:
Yeah just there so you can run update.php and make a backup for now. I adjusted the code so that if there's no updates, there's no backup step, it just jumps you to the "no updates needed" message.

Reply:
I removed this update hook to avoid confusion.